modules/ensemble/lib/widget/widget_registry.dart

```diff
@@ -175,6 +176,7 @@ class WidgetRegistry {
         ScrollableTabBar.type: () => ScrollableTabBar(),
         AppScroller.type: () => AppScroller(),
         LoadingContainer.type: () => LoadingContainer(),
+        InlineTimePicker.type: () => InlineTimePicker(),
```

Comment:
this is called `legacyWidgetMap` for a reason. See the comment in code - /// Legacy: To be moved to _registeredWidgets;

and use `_registeredWidgets`